packages/ui/src/components/NavMenu/NavMenu.tsx

```diff
@@ -160,7 +160,7 @@ const navMenuLink = tv({
       isSmallScreen: false,
       type: "link",
       class:
-        "text-white hover:text-primary-200 data-active:text-primary-200 hover:data-[icon=true]:text-primary-700 dark:data-active:text-primary-100 dark:hover:data-[icon=true]:text-primary-100",
+        "text-white hover:text-primary-200 data-active:text-primary-200 hover:data-[icon=true]:text-primary-700 dark:data-active:text-primary-200 dark:hover:data-[icon=true]:text-primary-200",
```

Comment:
The active hover state is too dark. Icon links now have the hover issue again :sob:

Reply:
Okay I think I got everything now with [c0c5c90](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13930/commits/c0c5c90f15ed9cfc396ff25a72bea193de254976) and [f0f0fb7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13930/commits/f0f0fb70b23d018ce7ef13b0b0e0a8717d4868d9)